include/psp2/promoterutil.h

```diff
@@ -13,12 +13,27 @@
 extern "C" {
 #endif
 
+/** Avalible types for ::ScePromoterUtilityImportParams **/
+typedef enum ScePromoterUtilityPackageType{
+	SCE_PKG_TYPE_VITA               = 0x0001,          //!< PSVita Apps
+	SCE_PKG_TYPE_PSM                = 0x0003,          //!< PlayStation Mobile
+}ScePromoterUtilityPackageType;
+	
 /** Parameters for scePromoterUtilityUpdateLiveArea() */
 typedef struct ScePromoterUtilityLAUpdate {
 	char titleid[12];  //!< Target app.
 	char path[128];    //!< Directory of extracted LA update data.
 } ScePromoterUtilityLAUpdate;
-
+	
+/** Parameters for scePromoterUtilityPromoteImport() */
+typedef struct ScePromoterUtilityImportParams{
+	char path[0x80]; //!< Install path usually (ux0:/temp/game) 
+	char titleid[0xC]; //!< Game titleid
+	ScePromoterUtilityPackageType type; //!< Package type
+	uint32_t attribute; //!< Additional Attributes
```

Comment:
you added an expected information for this variable in before review. why removed it that this is unimportant?
and other question, is this value output variable? `Appears to be 0x1 on PSM content but 0x00 on Vita` ?

Reply:
good catch, this is unacceptable

Reply:
its definitely not an output since the struct stays the same before/after the function being run.

i added it back- it was mainly just when i was changing text around to say its "Aditional attributes" i just removed it without thinking- should probably keep it there until its known that its acturally for.